p2p/discovery.py

```diff
@@ -125,25 +128,23 @@ def _get_handler(self, cmd: Command) -> Callable[[kademlia.Node, List[Any], byte
         else:
             raise ValueError("Unknwon command: {}".format(cmd))
 
-    def _get_max_neighbours_per_packet(self):
+    def _get_max_neighbours_per_packet(self) -> int:
         if self._max_neighbours_per_packet_cache is not None:
             return self._max_neighbours_per_packet_cache
         self._max_neighbours_per_packet_cache = _get_max_neighbours_per_packet()
         return self._max_neighbours_per_packet_cache
 
-    def connection_made(self, transport):
+    def connection_made(self, transport: Any) -> None:
```

Comment:
Can't we use an actual transport type here?

Reply:
Ah, good hint. It's one of these things that I skipped and never came back to. So this represents something like a `asyncio.DatagramTransport` right?

Reply:
Fixed by https://github.com/ethereum/py-evm/pull/947/commits/87d585801d1da8b99dbe60595299dab5734c671f